pypika/tests/test_aggregate.py

```diff
@@ -97,20 +106,15 @@ def test__case_with_field_is_not_aggregate(self):
 
         self.assertFalse(v.is_aggregate)
 
-    def test__case_with_single_aggregate_field_is_not_aggregate(self):
+    def test__case_with_single_aggregate_field_in_one_criterion_is_aggregate(self):
         v = (
             Case()
             .when(Field("foo") == 1, 1)
             .when(fn.Sum(Field("foo")) == 2, 2)
             .else_(3)
         )
 
-        self.assertFalse(v.is_aggregate)
-
-    def test__case_all_constants_is_aggregate_none(self):
-        v = Case().when(True, 1).when(False, 2).else_(3)
```

Comment:
Is this test no longer relevant?

Reply:
I think it makes no sense. It's not testing a real scenario, given the criterion is a constant. That's not how you'd use a CASE statement in SQL.

Reply:
The use case I can see for it is for renaming values e.g. "true" -> "foo", "false" -> "bar"

Reply:
But then you'd have a conditional. If field x is true then foo.

Reply:
Makes sense, thanks.